contributors/help-wanted.md

```diff
@@ -0,0 +1,138 @@
+<!-- source: https://github.com/kubernetes/community/blob/master/contributors/guide/help-wanted.md -->
+
+## Overview
+
+We use two labels to identify issues that have been specifically created or selected for new contributors: [help wanted](#help-wanted) and [good first
+issue](#good-first-issue). The `good first issue` label is a subset of the `help wanted`
+label, indicating that members have committed to providing extra assistance for
+new contributors. All `good first issue` items also have the `help wanted`
+label.
+
+We also have some [suggestions](#suggestions) for using these labels to help
+grow and improve our community.
+
+## Help Wanted
+
+Items marked with the `help wanted` label need to ensure that they meet these criteria:
+
+- **Low Barrier to Entry**
+  It should be easy for new contributors. Documentation on how that type of
+  change should be made should already exist.
+
+- **Clear Task**
+  The task is agreed upon and does not require further discussions in the
+  community. Call out if that area of code is untested and requires new
+  fixtures.
+
+  API and CLI behavior should be decided and included in the OP issue, for example: "The
+  new command syntax is `svcat unbind NAME [--orphan] [--timeout 5m]`", with
+  expected validations called out.
+
+- **Goldilocks priority**
+  The priority should not be so high that a core contributor should do it, but not too low that it
+  isn't useful enough for a core contributor to spend time reviewing it, answering
+  questions, helping get it into a release, etc.
+
+- **Up-To-Date**
+  Often these issues become obsolete and have already been completed, are no longer
+  desired, no longer make sense, or have changed priority or difficulty.
+
+<!-- TODO find a good one
+A good example of a Help Wanted issue description can be found here: [kubernetes/test-infra#21356 (comment)](https://github.com/kubernetes/test-infra/issues/21356#issuecomment-799972711).
```

Comment:
Same?

Reply:
This is enclosed inside html (or rather xml) multiline comment tags, that's why the `TODO` marker is still there.

Reply:
But yes, I am going to remove this.